src/ui/server/routes/insurance/export-contract/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { get, post } from '../../../test-mocks/mock-router';
+import { EXPORT_CONTRACT } from '../../../constants/routes/insurance/export-contract';
+import { get as exportContractRootGet } from '../../../controllers/insurance/export-contract';
+
+const { ROOT } = EXPORT_CONTRACT;
+
+describe('routes/insurance/export-contract', () => {
+  beforeEach(() => {
+    require('.'); // eslint-disable-line global-require
```

Comment:
do we need this?

Reply:
Unfortunately so, we have this in all UI route tests.